### PR TITLE
netutils/cjson: Enable cjson test

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Impact

New test case, but disabled by default

## Testing

Pass CI

### CODE REVIEW

File: netutils/cjson/Kconfig [hunk 1]
```diff
@@ -20,4 +20,16 @@ config NETUTILS_CJSON_VERSION
 	string "Version number"
 	default "1.7.12"
 
+config NETUTILS_CJSON_TEST
+	bool "enable cjson test"
```
Comment (pkarashchenko) [inline netutils/cjson/Kconfig:24]:
```suggestion
	bool "Enable cJSON test"
```

File: netutils/cjson/Kconfig [hunk 2]
```diff
@@ -20,4 +20,16 @@ config NETUTILS_CJSON_VERSION
...
+	default n
+
+if NETUTILS_CJSON_TEST
+
+config CJSON_UNITY_TEST_DIR
+	string "cjson test dir"
```
Comment (pkarashchenko) [inline netutils/cjson/Kconfig:30]:
```suggestion
	string "cJSON test directory"
```

File: netutils/cjson/Kconfig (hunk 1 above)
Comment (xiaoxiang781216) [reply netutils/cjson/Kconfig]: Done.

File: netutils/cjson/Kconfig (hunk 2 above)
Comment (xiaoxiang781216) [reply netutils/cjson/Kconfig]: Done.

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**